src/main/java/com/werfen/report/service/pdf/util/PdfExportService.java

```diff
@@ -1,53 +1,22 @@
 package com.werfen.report.service.pdf.util;
 
 import net.sf.jasperreports.engine.JRException;
+import net.sf.jasperreports.engine.JasperExportManager;
 import net.sf.jasperreports.engine.JasperPrint;
-import net.sf.jasperreports.pdf.JRPdfExporter;
-import net.sf.jasperreports.export.SimpleExporterInput;
-import net.sf.jasperreports.export.SimpleOutputStreamExporterOutput;
-import net.sf.jasperreports.pdf.SimplePdfExporterConfiguration;
-import net.sf.jasperreports.pdf.SimplePdfReportConfiguration;
 
 import java.io.ByteArrayOutputStream;
 import java.io.File;
 
 public class PdfExportService {
 
     public File exportToFile(JasperPrint jasperPrint, String filePath) throws JRException {
-        JRPdfExporter exporter = exportDocument(jasperPrint, new SimpleOutputStreamExporterOutput(filePath));
-
-        exporter.exportReport();
+        JasperExportManager.exportReportToPdfFile(jasperPrint, filePath);
         return new File(filePath);
     }
 
     public ByteArrayOutputStream exportToStream(JasperPrint jasperPrint) throws JRException {
         ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
-        SimpleOutputStreamExporterOutput simpleOutputStreamExporterOutput = new SimpleOutputStreamExporterOutput(outputStream);
-        JRPdfExporter exporter = exportDocument(jasperPrint, simpleOutputStreamExporterOutput);
-
-        exporter.exportReport();
+        JasperExportManager.exportReportToPdfStream(jasperPrint, outputStream);
         return outputStream;
     }
-
-    private JRPdfExporter exportDocument(JasperPrint jasperPrint, SimpleOutputStreamExporterOutput simpleOutputStreamExporterOutput) {
-        JRPdfExporter exporter = new JRPdfExporter();
-        exporter.setExporterInput(new SimpleExporterInput(jasperPrint));
-        exporter.setExporterOutput(simpleOutputStreamExporterOutput);
-        exporter.setConfiguration(getPdfReportConfiguration());
-        exporter.setConfiguration(getPdfExporterConfiguration());
-        return exporter;
-    }
-
-    private SimplePdfReportConfiguration getPdfReportConfiguration() {
-        SimplePdfReportConfiguration reportConfig = new SimplePdfReportConfiguration();
-        reportConfig.setSizePageToContent(true);
```

Comment:
There was certain configuration that has been lost due to this change: 
- set size page to content
- force line break policy
- allowed permissions hint

I think this PR shouldn't be for refactors and should be focused on a component upgrade, thus I would suggest to revert changes on this file and limit it to version upgrade.

Reply:
As we agreed, apart from running the tests  I reviewed the documents visually and there is no visual changes apart from the ones derived from the changes in the fonts library (not related with the changes in the implementation).